Src/Support/IntegrationTests/PerCallAuthTests.cs

```diff
@@ -161,9 +169,15 @@ public void OverridesServiceCredentialWithPerCallCredential_Mixed_Query_Header()
         [Fact]
         public void OverridesServiceCredentialWithPerCallCredential_TwoCallCredentials()
         {
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Let's just have a single disable and restore per method (so remove the middle 4 pragmas)